.circleci/config.yml

```diff
@@ -12,76 +12,183 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
   sfchangecase: salesforce/change-case-management@3
+  slack: circleci/slack@3.4.2
 
-_refs:
```

Comment:
Moved YAML anchors into commands as per circleci recommendations. These have been made to be flexible depending on how we need to use them. Some of the new commands will be candidates for a new salesforce orb that we and other teams can call in the future.